doc-content/jbpm-docs/src/main/asciidoc/CoreEngine/deployment-synchronization-con.adoc

```diff
@@ -0,0 +1,43 @@
+[id='deployment-synchronization-con_{context}']
+= Deployment synchronization
+
+ifdef::DROOLS,JBPM,OP[]
+Prior to jBPM 6.2, jbpm services did not have a deployment store by default.
+When embedded in jbpm-console/kie-wb, they utilized the sistem.git VFS repository to preserve deployed units across server restarts.
+While that works fine, it comes with some drawbacks:
+
+* not available for custom systems that use services
+* requires complex setup in cluster - zookeeper and helix
+
+With version 6.2 jbpm services come with a deployment synchronizer
+endif::DROOLS,JBPM,OP[]
+ifdef::PAM,DM[]
+{PROCESS_ENGINE} services include a deployment synchronizer
```

Comment:
Change to {PROCESS_ENGINE_CAP} if begins sentence. But it's probably good to try to avoid this cap variable anyway, unless you really need it. Issue is that for community it becomes "JBPM engine" (instead of "jBPM engine"), which isn't terrible, but is more for titles when you can't avoid it. If you can avoid it in prose, by using "The {PROCESS_ENGINE}" or whatever, probably best.